src/tools/sdk/sdk_builds.dart/lib/src/dart_downloads.dart

```diff
@@ -36,7 +36,7 @@ class DartDownloads {
     return new DartDownloads._(client);
   }
 
-  Future<Uri> getDownloadLink(
```

Comment:
Hmmm... "download" sounds like a verb, but I can't think of a better name. (In fact, I thought all function names were supposed to start with verbs... but TIL [that's not true](http://localhost:4000/guides/language/effective-dart/design#prefer-a-noun-phrase-or-non-imperative-verb-phrase-for-a-function-or-method-if-returning-a-value-is-its-primary-purpose).)

Reply:
That's true, this name does make it sound like a verb. How about `createDownloadUrl`?

Reply:
sgtm